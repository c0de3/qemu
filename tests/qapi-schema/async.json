{ 'command': 'screendump-async', 'data': {'filename': 'str'}, 'async': true }
