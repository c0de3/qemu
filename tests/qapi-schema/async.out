enum QType ['none', 'qnull', 'qint', 'qstring', 'qdict', 'qlist', 'qfloat', 'qbool']
    prefix QTYPE
object q_empty
object q_obj_screendump-async-arg
    member filename: str optional=False
command screendump-async q_obj_screendump-async-arg -> None
   gen=True success_response=True boxed=False async=True
